tsconfig.json

```diff
@@ -15,9 +15,9 @@
     "noImplicitReturns": true,
     "noImplicitThis": true,
     "noImplicitAny": false,
-    "strictNullChecks": true,
+    "strictNullChecks": false,
```

Comment:
:-(
So there's no better solution? It's a serious regression in code quality enforcement.

Is there a way to deport this null-check / unused params checks rules to the linter so that we still check our own code and we aren't impacted by failures in 3rd party code?

Reply:
 I'm open for suggestions. But this is the only way to workaround the blocker.

Reply:
Can't we just use `any` on the input params?

Reply:
This change is already reverted.
So, this PR doesn't modify the tsconfig.json